aries-site/src/components/feedback/Feedback.js

```diff
@@ -0,0 +1,124 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  Form,
+  Heading,
+  Layer,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { FormClose } from 'grommet-icons';
+
+export const Feedback = ({
+  children,
+  modal,
+  onChange,
+  onClose,
+  onEsc,
+  onSubmit,
+  onReset,
+  show,
+  title,
+  value,
+  isSucessful,
+}) => {
+  const size = useContext(ResponsiveContext);
+
+  let content = (
+    <Box
+      fill="vertical"
+      overflow="auto"
+      width={!['xsmall', 'small'].includes(size) ? 'medium' : undefined}
+      pad="medium"
+      flex
+    >
+      <Identifier onClose={onClose} title={title} modal={modal} />
+      <Box gap="medium" margin={{ vertical: 'small' }}>
+        <Form
+          value={value}
+          onChange={onChange}
+          onSubmit={onSubmit}
+          onReset={onReset}
+          method="post"
+          validate="submit"
+        >
+          {children}
+          <>
+            {!isSucessful ? (
+              <Box
+                pad={{ top: 'medium' }}
+                justify="end"
+                gap="medium"
+                direction="row"
+              >
+                <Button onClick={onClose} label="Cancel" />
+                <Button
+                  onSubmit={onSubmit}
```

Comment:
I am able to hit submit without filling out any of the fields, maybe we should consider making at least one of them mandatory

Reply:
yeah thats kinda the direction Megan wanted to go was not to make any of this required but I can bring that up to her or maybe we have the thumbs up as default so then on submit its at least that.

Reply:
agree ☝️ we're not going to default any response